Berksfile

```diff
@@ -0,0 +1,6 @@
+source 'https://supermarket.getchef.com'
+
+# We want to be explicit, since we don't explicitly include our
+# packages, except when testing
+cookbook 'build-essential', '1.4.2'
+cookbook 'git', '3.1.0'
```

Comment:
@k-kinzal I don't think so, it worked fine without adding php as a dependency to the Berksfile.